### PR TITLE
feat: add support for Kubernetes v1.20.0-rc.0

### PR DESCRIPTION
**Pull request by mboersma**
**Reason for Change**:

See https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#changelog-since-v1200-beta2

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [x] No
- [ ] Yes

**Requirements**:

- [x] Kubernetes artifacts built and pushed by Azure Pipelines
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by jackfrancis** [comment]
@mboersma @cpuguy83 FYI, this PR fixed the DNS check thing:

https://github.com/Azure/aks-engine/pull/4080

tl;dr running nslookup without a targeted option (like `-type=A`) takes a long time and the livenessProbe foo times out. This is probably a change in the underlying livenessProbe implementation and this change should be investigated, but we can also be good livenessProbe citizens and optimize the way we use nslookup (I assume the long time is due to ipv6 DNS requests by default, which we don't want anyways).

cc @aramase if you're aware of any changes where livenessProbe will throw a context deadline timeout exception if a command takes a long time (~a few seconds)

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Comment by mboersma** [comment]
> this PR fixed the DNS check thing

Wow, we got lucky for a change. 🍀
